tests/robustness/traffic/kubernetes.go

```diff
@@ -322,13 +321,8 @@ func (s *storage) Count() int {
 func (s *storage) PickRandom() (key string, rev int64) {
 	s.mux.RLock()
 	defer s.mux.RUnlock()
-	n := rand.Intn(len(s.keyRevision))
-	i := 0
 	for k, v := range s.keyRevision {
```

Comment:
Let's keep the `PickRandom` function random and fair. Picking first element from map, might seem random, but in reality it will return the same element everytime if map was not changed. I think it's important to have this function to have some equal distribution.

Reply:
Looks random to me: https://go.dev/play/p/HjoOidSuBOq

https://go.dev/ref/spec#For_range:
> The iteration order over maps is not specified and is not guaranteed to be the same from one iteration to the next.

You are right. It is not guaranteed to change. I will remove it when splitting the changes.